_includes/domain-search.html

```diff
@@ -27,7 +27,7 @@
                 <div class="usa-alert usa-alert--success usa-alert--slim" role="alert">
                     <div class="usa-alert__body">
                         <p class="usa-alert__text">
-                        <b>${requested_domain_display_string}</b> is available!
+                        <b>${requested_domain_display_string}</b> is available !
```

Comment:
nonblocking: Is this space before the ! supposed to be there?

Reply:
Great catch -- fixed in [this commit](https://github.com/cisagov/getgov-home/pull/136/commits/1c09bdcd73032c187b6de0a89f44e131dcc10ffb)!